awx/main/access.py

```diff
@@ -2429,6 +2429,11 @@ def filtered_queryset(self):
     def can_add(self, data):
         if not JobLaunchConfigAccess(self.user).can_add(data):
             return False
+        if not data:
```

Comment:
Without this `HTTP OPTIONS /api/v2/schedules/` always is missing `"POST"` for non-superusers.